### DISCUSSION TITLE
Only half of document gets ingested

### DISCUSSION THREAD

**Discussion by mscherotter** [microsoft/AI-Chat-App-Hack#40] [Q&A]
I have setup my RAG using Azure Search and with a skillset using th eAzureOpenAIEmbeddingSkill and my Custom Web Skill for chunking PDF Files.  I started with the vectorize wizard with azure search.  I am noticing that the text that I get is only part of the text in the document (the first half).  How do I increase the ingest size of the documents?

**Comment by pamelafox** [comment]
@srbalakr Do you know what might be happening here?

**Reply by srbalakr** [reply]
@mscherotter could you share the definition of custom web skill. Also when you choose wizard in azure search, we would define the chunking skill. Could you share the screen shot, I would like to know how do you use your custom web skill.
